src/Instance/InstanceDetails/InstanceTitleData/AlternativeTitlesList.js

```diff
@@ -21,18 +21,30 @@ const getColumnMapping = intl => ({
   type: intl.formatMessage({ id: 'ui-inventory.alternativeTitleType' }),
   title: intl.formatMessage({ id: 'ui-inventory.alternativeTitle' }),
 });
-const formatter = {
-  type: item => item.type || noValue,
-  title: item => item.alternativeTitle || noValue,
-};
 const columnWidths = {
   type: '25%',
   title: '75%',
 };
 
+const getTitleItem = (item, segment, source) => {
```

Comment:
@BogdanDenis this looks exactly the same as in [InstanceSubjectView.js](https://github.com/folio-org/ui-inventory/pull/1977/files#diff-b4e449d19d4ce4ec1cea18f63bee488722446eb63a3cb5c1e66f91f13a869fbd)

Perhaps we could extract it so it's reused in both places?

Reply:
That's a good point, I've moved this logic to `<ControllableDetail>` component